src/Entity/Contributor.php

```diff
@@ -58,7 +58,6 @@ class Contributor implements ImageUploadable
      * @var string
      *
      * @ORM\Column(name="intro", type="string", length=255, nullable=true)
-     * @Assert\Length(max="255")
```

Comment:
You could keep the length assertion but make it longer.

Reply:
do you know what command to use to generate automatic migration ? migrations:diff generate something wrong

Reply:
```
doctrine:migrations:diff
```

... is the right command, it generate as lot's of "noise"' -- which is not normal and should be fixed sometimes -- which you should strip down.